src/index.md

```diff
@@ -2,14 +2,46 @@
 
 # Renovate documentation
 
-Renovate is an Open Source tool to automate:
+Automated dependency updates.
+Multi-platform and multi-language.
 
-- Detecting dependencies in a repository (Open Source and private/closed source)
-- Checking if there are dependency updates
-- Creating commits and Merge/Pull Requests to update dependencies
-- Showing the release notes
+## Why use Renovate?
 
-## Renovate's purpose
+- Get pull requests to update your dependencies and lock files
+- Reduce noise by scheduling when Renovate creates PRs
+- Renovate finds relevant package files automatically, including in monorepos
+- You can customize the bot's behavior with configuration files
+- Share your configuration with ESLint-like config presets
+- Get replacement PRs to migrate from a deprecated dependency to the community suggested replacement (npm packages only)
+- Open source
+
+## Supported Platforms
+
+Renovate works on these platforms:
+
+- GitHub (.com and Enterprise Server)
+- GitLab (.com and CE/EE)
+- Bitbucket Cloud
+- Bitbucket Server
+- Azure DevOps
+- Gitea
+
+## Who Uses Renovate?
+
+Renovate is used by:
+
+![Renovate Matrix](https://app.renovatebot.com/images/matrix.png)
+
+## The Renovate Approach
+
+We believe everyone benefits from automation, whether it's a little or a lot.
+This means that Renovate:
+
+- Adapts to your workflow
+- Allows you to configure its behavior
+- Will autodetect settings where possible
+
+## Renovate's Purpose
```

Comment:
Do we really _need_ the `The Renovate Approach` and `Renovate's Purpose` sections? I think the homepage would look better if we removed these sections, and let the main `Why use Renovate?` section entice the reader.

Reply:
Yes, we can remove